### PR TITLE
Fix yarpmanager

### LINKED ISSUE #1536: yarpmanager error when yarpserver is not found is not clear
When the yarpserver is not found, the yarpmanager prints the following message:
~~~
[ERROR]Manager::Cannot contact the NameClient 
~~~
As this is a common error for newbies, I propose to at least add a reference to `yarpserver` in the message (for a newbie, knowing that the NameClient is the `yarpserver` is not obvious). 

cc @Nicogene

### PR DESCRIPTION
**Pull request by Nicogene**
Minor fixes to the `ClusterWidget`.
It fixes #1536 .

Tested on `iCubGenova01` setup.
Please review code.

### CODE REVIEW

File: src/yarpmanager/src-manager/clusterWidget.cpp [hunk 1]
```diff
@@ -477,6 +485,20 @@ bool ClusterWidget::checkNode(const string &name)
         portname = "/" + portname;
     }
 
+    if (!NetworkBase::checkNetwork(2.0))
+    {
+        yError()<<"Manager::yarpserver is not running...";
```
Comment (traversaro) [inline src/yarpmanager/src-manager/clusterWidget.cpp]: Personal comment: I am not a big fan of the `...` at the error of the messages, they do not seem very professional to me. 
Second comment: why `Manager` at the beginning of the message? If this is the message of the `yarpmanager` executable, I would put the `yarpmanager` string for clarity. Furthermore, why you use the `::` as a separator, that seems to hint to C++ namespaces? I would just write `yarpmanager: yarpserver is not running.` .

**Comment by Nicogene** [comment]
@drdanz it is ready to be merged.

File: src/yarpmanager/src-manager/clusterWidget.cpp [hunk 2]
```diff
@@ -477,14 +485,28 @@ bool ClusterWidget::checkNode(const string &name)
...
+    if (!NetworkBase::checkNetwork(2.0))
+    {
+        yError()<<"ClusterWidget: yarpserver is not running";
+        return false;
+    }
+
+    yarp::profiler::NetworkProfiler::PortDetails dummy;
+    if (! yarp::profiler::NetworkProfiler::getPortDetails(portname, dummy))
+    {
+        yError()<<"ClusterWidget: port"<<portname<<"is not responding";
```
Comment (traversaro) [inline src/yarpmanager/src-manager/clusterWidget.cpp]: `port"<<portname<<"is` --> `port "<<portname<<" is`?

Comment (Nicogene) [reply src/yarpmanager/src-manager/clusterWidget.cpp]: `yError`,`yInfo`, `yWarning` add automatically the spaces :wink:

Comment (traversaro) [reply src/yarpmanager/src-manager/clusterWidget.cpp]: I had **no idea** of this.

**Comment by Nicogene** [comment]
@damn1 found an issue in `Network::checkNetwork()` that goes in deadlock for 1 second if yarp is compiled without ACE.
Let's investigate this issue before merging this PR.
@damn1  could you please open an issue ?

**Comment by Nicogene** [comment]
Waiting #1636 has been fixed before merging

**Comment by Nicogene** [comment]
Rebased and ready to be merged

**Comment by drdanz** [comment]
Merged, thanks.

**Pull request merged**